CHANGELOG.md

```diff
@@ -1,4 +1,66 @@
-# Change Log
+# Changelog
+
+## [v25.0.0](https://github.com/w3c/respec/tree/v25.0.0) (2019-12-30)
+
+[Full Changelog](https://github.com/w3c/respec/compare/v24.38.1...v25.0.0)
+
+**Implemented enhancements:**
+
+- Why do some specs end up with a class="informative" on body? [\#2580](https://github.com/w3c/respec/issues/2580)
+- Warn when superclass is not linked. [\#1108](https://github.com/w3c/respec/issues/1108)
+- Implement bikeshed's dfn contract [\#685](https://github.com/w3c/respec/issues/685)
+- Encapsulating the content into a \<main\> element? [\#479](https://github.com/w3c/respec/issues/479)
+- Link button in IDL definitions [\#444](https://github.com/w3c/respec/issues/444)
+- Add the possibility to completely replace the "status of this document" section [\#113](https://github.com/w3c/respec/issues/113)
```

Comment:
If issue contained "enhancement" label, they show as implemented even when we didn't implement them.

Reply:
That's ok... the CHANGELOG is just a guide for more details.